src/gesture_handler.rs

```diff
@@ -0,0 +1,22 @@
+use crate::{configuration::Gesture, gesture_listener::GestureListener};
+use input::event::gesture::*;
+
+
+
+pub struct GestureEventHandler {
+    listeners: Vec<Box<dyn GestureListener>>,
+}
+
+impl GestureEventHandler {
+    pub fn new(gestures: Vec<Gesture>) -> Self {
+        let listeners: Vec<Box<dyn GestureListener>> = gestures.into_iter().map(|gesture| Gesture::listen(gesture)).collect();
```

Comment:
I think this can be simplified, but I can't test because I am on Mac:
```suggestion
        let listeners: Vec<Box<dyn GestureListener>> = gestures.into_iter().map(Gesture::listen).collect();
```